joda-time/src/main/java/org/joda/time/chrono/ISOYearOfEraDateTimeField.java

```diff
@@ -56,8 +56,9 @@ public DurationField getRangeDurationField() {
     @Override
     public int get(long instant) {
         int year = getWrappedField().get(instant);
-        return year < 0 ? -year : year;
+        return year <  1 ? -year : year;
```

Comment:
Is this change actually required?

Reply:
No, please change just the formatting not the logic.

@Mariam22-hub please address this in the next 2-3 days, or we'll assign to someone else.